astro/src/content/docs/extend/code/password-hashes/custom-password-hashing.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Custom Password Hashing (Password Encryptors)
+title: Custom Password Hashing
```

Comment:
I'm getting an error when trying to view this page
http://localhost:3000/docs/extend/code/password-hashes/custom-password-hashing

Error: Cannot find module '@fusionauth/astro-components' imported from ....

Reply:
You need to run `npm install` to pick up the library

Reply:
Makes sense.  Thx.